concrete/src/Http/Middleware/CookieMiddleware.php

```diff
@@ -41,7 +41,7 @@ public function process(Request $request, DelegateInterface $frame)
             $response->headers->clearCookie($cookie, DIR_REL . '/');
         }
 
-        $cookies = $this->cookies->getCookies();
+        $cookies = $this->cookies->getNewCookies();
```

Comment:
If there is both a `getNewCookies` and `getAllCookies` method, shouldn't this be `getAllCookies` ?

Reply:
The former `getCookies` method returned only the cookies set by PHP code. I renamed it to `getNewCookies` to make it clear. And for BC I kept the old method name.
Here we only have to create new cookies (the browser already has the ones we receive from Request, in the Response we just add thr new ones).